src/Stratis.Bitcoin.Features.RPC/RPCFeature.cs

```diff
@@ -121,8 +115,10 @@ public override Task InitializeAsync()
     /// </summary>
     public static class FullNodeBuilderRPCExtension
     {
-        public static IFullNodeBuilder AddRPC(this IFullNodeBuilder fullNodeBuilder)
+        public static IFullNodeBuilder AddRPC(this IFullNodeBuilder fullNodeBuilder, Action<RpcSettings> defaultArgs = null)
         {
+            var options = new RpcSettings(fullNodeBuilder.NodeSettings, defaultArgs);
```

Comment:
We had this before and we moved away from this approach (we can just override values in the args)

Reply:
We don't want to override the args - we want to inject defaults that can be overridden by the args.